yarn.lock

```diff
@@ -908,7 +908,7 @@ __metadata:
     aws-cdk-lib: ~2.80.0
     aws-sdk: ^2.1464.0
     aws-sdk-mock: ^5.8.0
-    axios: ^1.6.2
+    axios: ^1.6.5
```

Comment:
Don't `^1.6.2` and `^1.6.5` end up with the same version?

Reply:
With `^1.6.2` follow-redirects was resolving to `1.15.2`, bumping axios did the trick to have follow-redirects resolve to a patched version